platform/darwin/scripts/style-spec-overrides-v8.json

```diff
@@ -1,4 +1,10 @@
 {
+  "light-cocoa-doc": "An `MGLLight` object represents the light source for extruded geometries in `MGLStyle`.",
+  "light": {
+      "position": {
+        "doc": "Position of the `MGLLight` source relative to lit (extruded) geometries, in a `MGLSphericalPosition` struct [radial coordinate, azimuthal angle, polar angle] where radial indicates the distance from the center of the base of an object to its light, azimuthal indicates the position of the light relative to 0° (0° when `MGLLight.anchor` is set to `MGLLightAnchorViewport` corresponds to the top of the viewport, or 0° when `MGLLight.anchor` is set to `MGLLightAnchorMap` corresponds to due north, and degrees proceed clockwise), and polar indicates the height of the light (from 0°, directly above, to 180°, directly below)."
```

Comment:
> [radial coordinate, azimuthal angle, polar angle]

This bit is unnecessary – it’s in the style specification because the JSON syntax literally requires an array with this ordering. Per https://github.com/mapbox/mapbox-gl-native/pull/9260#discussion_r121874710, everything from “where” onwards can be moved to the MGLLight documentation, since developers who want to learn how to use MGLLight might go there first.